test/e2e/apps/wait.go

```diff
@@ -0,0 +1,60 @@
+/*
```

Comment:
Why don't we move functions into ` test/e2e/apps/deployment.go` directly?
These functions are used in ` test/e2e/apps/deployment.go` only.

Reply:
Ah I am asking @SataQiu  at the above

Reply:
I think it's okay to move functions into `test/e2e/apps/deployment.go` @oomichi 
Just don't recommend moving these functions into `test/e2e/apps/framework.go`.
Can you update it again? @tanjunchen

Reply:
Sure @oomichi @SataQiu Thanks advice